src/gpl/src/mbff.h

```diff
@@ -0,0 +1,72 @@
+#include <vector>
```

Comment:
Needs the standard BSD header (see any other file).  You can assign the copyright to "Precision Innovations Inc."